packages/technical-features/kubectl-versions/index.ts

```diff
@@ -0,0 +1,3 @@
+import kubectlVersionsInjectable from "./src/versions.injectable";
+
+export { kubectlVersionsInjectable };
```

Comment:
What is the benefit of exporting an injectable?

Reply:
Downsides for exporting `injectable`:
- What would happen when someone decides to `register` or even `deregister` the injectable. This shouldn't be allowed
- Usage would depend from the implementation and not from the contract
- It would be harder to change our mind with technical details because they would have been leaked to the usage

Therefore, I would suggest to always `export` the `injectionToken` instead. This doesn't have same downsides and corner cases.